src/modules/profile/sagas.ts

```diff
@@ -115,30 +115,22 @@ export function createProfileSaga({ peerUrl }: CreateProfileSagaOptions) {
     address: string,
     changes: Partial<Avatar>
   ) {
-    const entity: ProfileEntity = yield call(
-      [entities, entities.getProfileEntity],
-      address
-    )
+    const profile: Profile = yield call([peerApi, 'fetchProfile'], address)
```

Comment:
`fetchProfile` seems to have some kind of cache. We might want to remove it because it might break the user experience if they want to update more than once their profiles, right?